src/gui/generalsettings.ui

```diff
@@ -222,7 +222,7 @@
         <item>
          <widget class="QPushButton" name="debugArchiveButton">
           <property name="text">
-           <string>Create Debug Archive...</string>
+           <string>Create Debug Archive ...</string>
```

Comment:
My fault there... Could you also please switch it to a proper ellipsis instead of three dots like I stupidly did?

Reply:
I changed 😉.

Reply:
Thanks!